### ISSUE TITLE
[BUG] Blips missing

### ISSUE THREAD

**Issue by TonybynMp4** [qbcore-framework/qb-hud#126]
**Describe the bug**
The minimap.gfx streamed is missing blips (exemple shown in the screenshots)
The Blip im using particular is ID 740 wich is the Gta online arcade blip

I think it might come from the minimap.gfx being from an older version of the game (pre b2060) and therefore missing blips

**To Reproduce**
Add a blip with the ID 740 
Look at the blip While not streaming qb-hud (so not having it start on the server, stopping the resource while the server is already on will not stop it from streaming, at least without rejoining i think)
and then simply restart the server with qb-hud and the blip should've changed

**Expected behavior**
It should be the blip in the first screenshot

**Screenshots**
Without the minimap.gfx
![image](https://user-images.githubusercontent.com/97451137/156861202-4db7bb01-5c9c-49ee-8c95-10ba07cddf93.png)

With the minimap.gfx
![image](https://user-images.githubusercontent.com/97451137/156861542-4a39a9ec-8b57-43c2-b602-f8768955aaef.png)

**Questions (please complete the following information):**
 - When you last updated: Today
 - Are you using custom resource? which ones? i am using a few but i think i pretty much spotted the problem since it happened only after i updated it and simply deleting the .gfx fixed it (while breaking the minimap by showing the vanilla health/armor bars tho)
 - Have you renamed `qb-` to something custom? nop

**Comment by TonybynMp4** [comment]
The fix for that shouldn't be too hard, 
just update to a newer minimap.gfx and edit it to remove what the old one removes, 
i just don't know how the f*ck to do it lol.

**Comment by MonkeyWhisper** [comment]
Issue confirmed. 
![image](https://user-images.githubusercontent.com/82112471/172121193-a2ffaea3-5517-419e-b64b-5dab367f5764.png)

**Comment by ZumieFiasco** [comment]
Has there been any update on this? I just came to report this. But yes, the minimap.gfx is outdated and blips from newer builds are unable to load properly. I was going to resort to simply deleting the minimap.gfx file but in doing so the old health bar and armor bar status show up which obviously is not the desired result. Would like to be able to use up to date blips from new builds! : )

**Comment by TonybynMp4** [comment]
> Has there been any update on this? I just came to report this. But yes, the minimap.gfx is outdated and blips from newer builds are unable to load properly. I was going to resort to simply deleting the minimap.gfx file but in doing so the old health bar and armor bar status show up which obviously is not the desired result. Would like to be able to use up to date blips from new builds! : )

Sadly it seems the devs making the resource aren't gonna fix this, tried asking project sloth since they use the same **broken** .gfx but they won't do it so idk i've lost faith at this point.

**Comment by TonybynMp4** [comment]
@ZumieFiasco use #154 :)
